comm/src/main/scala/coop/rchain/comm/transport/TcpTransportLayer.scala

```diff
@@ -200,7 +202,7 @@ class TcpTransportLayer(host: String, port: Int, cert: String, key: String, maxM
                        new RuntimeException("TransportLayer server is already started")
                      )
                    case _ =>
-                     val parallelism = Runtime.getRuntime.availableProcessors()
+                     val parallelism = Runtime.getRuntime.availableProcessors() + 1
```

Comment:
Why +1?

Reply:
@goral09 suggested to do `max(number cores, 2)` because it's best practice. I thought that +1 would achieve the same thing.

Reply:
@sebtomba If you have 4 cores, the original version will set `2` and yours `5`.

Reply:
The following article gives a good explanation on why people often use the `+ 1`:
https://medium.com/@sergeykuptsov/how-it-works-in-java-thread-pool-2a15dac75620
Basically to still have maximum parallelism in the case one thread takes a break.